packages/router/src/active-route-loader.tsx

```diff
@@ -126,7 +126,7 @@ export const ActiveRouteLoader = ({
               location,
             },
           }))
-          setRenderedChildren(children)
+          setRenderedChildren(children ?? module.default)
```

Comment:
Mind the explaining this change?

Reply:
Thanks for asking. Instead of giving an explanation here I added a comment to the code. Hopefully that'll make thing more clear for you, and for all future readers of the code.